### PR TITLE
Several Minor Fixes to various pages

### PR DESCRIPTION
**Pull request by Vimal-Raghubir**
### Proposed changes

- Fixed a typo in kuber-primer.md where the name of the file was incorrectly written as k8-app1.yml when it should be k8-app.yml

- Restructured a sentence in docker-overview.md because of grammatical issues.

- Fixed incorrect link when clicking the Open an Issue hyperlink to redirect to the correct GitHub Issue template.

### Related issues (optional)
Refer to issues: #6742, #6699, and #6686 respectively for the above mentioned changes.

### CODE REVIEW

File: 404.md [hunk 1]
```diff
@@ -11,6 +11,6 @@ track, maybe you found a broken link. Who knows?
 
 Either way, here's a couple of options for you:
 
-* [Open an issue](https://github.com/docker/docker.github.io/issues/new?title=I%20found%20a%20broken%20link&body=%3CDescribe%20how%20you%20got%20there%3E)
+* [Open an issue](https://github.com/docker/for-win/issues/new)
```
Comment (joaofnfernandes) [inline 404.md]: I think you are trying to address #6686 with this change but I think that issue refers to a problem in the `docker/for-win` repository, and not the documentation.

This change will make the docs 404 page send users to the docker for Windows repository, which I don't agree with.
Can you revert this back to how it was?

File: docker-cloud/migration/kube-primer.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ This command deploys a Docker application, named `test-app`, from a YAML configu
 $ docker stack deploy -c app1.yml test-app
 ```
 
-This command deploys a Kubernetes application from a YAML manifest file called `k8s-app1.yml`:
+This command deploys a Kubernetes application from a YAML manifest file called `k8s-app.yml`:
```
Comment (joaofnfernandes) [inline docker-cloud/migration/kube-primer.md]: This change is not correct. Line 94 tells users to create a file names `app1.yml`, so lines 94, 97, 100 should use `app1.yml`.

**Comment by Vimal-Raghubir** [comment]
Had to recreate PR since I made the changes from GitHub website and there is no revert functionality yet. Please see new PR #6748.

File: docker-cloud/migration/kube-primer.md (hunk 2 above)
Comment (Vimal-Raghubir) [reply docker-cloud/migration/kube-primer.md]: Please see new PR below for the suggested change. :+1:

File: 404.md (hunk 1 above)
Comment (Vimal-Raghubir) [reply 404.md]: I removed this change in the new PR below.

**Pull request closed without merging**